packages/rocket-uploads3-store-event-aws-infrastructure/yarn.lock

```diff
@@ -0,0 +1,2043 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
Notice that while this PR was open, we've switched to npm, so please rebase it to master and delete the yarn.lock file

Reply:
done :)